tests/foreman/longrun/test_inc_updates.py

```diff
@@ -212,3 +214,108 @@ def test_positive_noapply_api(
         outval['action']
         == 'Incremental Update of 1 Content View Version(s) with 1 Package(s), and 1 Errata'
     )
+
+
+@pytest.mark.tier3
+def test_positive_incremental_update_time(module_target_sat, module_sca_manifest_org):
+    """Incremental update should not take a long time.
+
+    :id: a9cdcc58-2d10-42cf-8e24-f7bec3b79d6b
+
+    :steps:
+        1. Setup larger rh repositories; rhel8 baseOS, rhst8, rhsc8.
+        2. Create content view and add repos, sync and wait.
+        3. Publish a content view version with all content.
+        4. Using hammer, perform incremental update with errata, on that new version.
+            - Log the duration of the incremental update
+        5. Publish the full content-view, with added incremental version.
+            - Log the duration of the content-view publish
+
+    :expectedresults:
+        1. Incremental update takes a short amount of time.
+        2. Incremental update takes less time than full content-view publish,
+            or the time taken for both was close (within 20%).
+
+    :BZ: 2117760, 1829266
```

Comment:
```suggestion
    :BZ: 2117760, 1829266
    
    :customerscenario: true
```
This seems to be a customer-case scenario.